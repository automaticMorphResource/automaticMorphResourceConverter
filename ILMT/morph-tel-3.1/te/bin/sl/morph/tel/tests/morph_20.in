1	ikkada	unk	
1	vanyaprANulu	unk
1	annI	unk	
1	jUlono	unk	
1	,	unk	
1	pArkulono	unk	
1	baMXiMcinatlugA	unk	
1	kAka	unk
1	..	unk	
1	svecCagA	unk	
1	wiruguwU	unk	
1	uMtAyi	unk	
1	.	unk	
