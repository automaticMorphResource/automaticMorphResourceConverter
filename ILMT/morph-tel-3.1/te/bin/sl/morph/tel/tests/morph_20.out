1	ikkada	unk	<fs af='ikkada,n,,sg,,d,0,0'>|<fs af='ikkada,n,,sg,,o,ti,ti'>
2	vanyaprANulu	unk	<fs af='vanyaprANi,n,,pl,,d,0,0'>|<fs af='vanyaprANi,n,,pl,,d,0,0'>
3	annI	unk	<fs af='annI,avy,,,,,0,0_avy'>
4	jUlono	unk	<fs af='jU,n,,sg,,,lo,lo_o'>
5	,	unk	<fs af='&comma;,punc,,,,,,'>
6	pArkulono	unk	<fs af='pArku,n,,sg,,,lo,lo_o'>|<fs af='pArku,n,,sg,,,lo,lo_o'>
7	baMXiMcinatlugA	unk	<fs af='baMXiMcu,v,any,any,any,,ina_atlugA,ina_atlugA'>
8	kAka	unk	<fs af='kAka,avy,,,,,,'>|<fs af='kAka,n,,sg,,d,0,0'>|<fs af='kAka,n,,sg,,o,ti,ti'>|<fs af='kAkaM,n,,sg,,o,ti,ti'>|<fs af='avvu,v,any,any,any,,aka,aka'>
9	..	unk	<fs af='&dot;,punc,,,,,,'>
10	svecCagA	unk	<fs af='svecCa,n,,sg,,,gA,gA'>
11	wiruguwU	unk	<fs af='wirugu,v,any,any,any,,wU,wU'>
12	uMtAyi	unk	<fs af='uMdu,v,n,pl,3,,wA,wA'>
13	.	unk	<fs af='&dot;,punc,,,,,,'>
